packages/react-native-editor/CHANGELOG.md

```diff
@@ -11,46 +11,56 @@ For each user feature we should also add a importance categorization label  to i
 
 ## Unreleased
 
+-   [*] [Image block] Fix missing translations [#37956]
+
 ## 1.69.0
+
```

Comment:
I noticed that an extra line has been introduced to all sections of the changelog, @jhnstn is this expected?

Reply:
yeah I just noticed that, it was auto formatted by VScode. But I just noticed half of the file has the extra whitespace so I'm thinking keep the extra space. Happy to revert that if you think it's better to have no space.

Reply:
I saw that other changelog files have this format ([example reference](https://raw.githubusercontent.com/WordPress/gutenberg/trunk/packages/rich-text/CHANGELOG.md)), so looks great to me leaving it this way 👍 .